### PR TITLE
drivers: adc: ltc2451: Add ltc2451 resolution to config

### PR DESCRIPTION
**Pull request by ethan-duckett-brill**
Adds resolution to config struct

### CODE REVIEW

**Review by aasinclair — COMMENTED** [review]
What is the resolution value used for?

**Comment by ethan-duckett-brill** [comment]
> What is the resolution value used for?

I wasn't entirely sure whether the resolution value was useful enough to be upstreamed, but locally we use it in some calculations so I wondered whether it would be useful for others.

File: drivers/adc/adc_ltc2451.c [hunk 1]
```diff
@@ -18,6 +18,7 @@ LOG_MODULE_REGISTER(ltc2451, CONFIG_ADC_LOG_LEVEL);
 struct ltc2451_config {
 	struct i2c_dt_spec i2c;
 	uint8_t conversion_speed;
+	int resolution;
```
[the remark this answers is not in the document]
Comment (ethan-duckett-brill) [reply drivers/adc/adc_ltc2451.c:21]: Ah apologies, we have it defined in the header file locally. We use it to for example in a calculation to convert ADC counts to Volts etc. I'd created and upstreamed the original driver but we've been using our local version until now.

[the remark this answers is not in the document]
Comment (ubieda) [reply drivers/adc/adc_ltc2451.c:21]: I agree with @aasinclair

**Pull request closed without merging**